addons/stock_account/models/product.py

```diff
@@ -73,11 +73,6 @@ def _is_cost_method_standard(self):
     def _set_cost_method(self):
         return self.write({'property_cost_method': self.cost_method})
```

Comment:
Let us me think that if we change on the category, we should not change the products which have property_cost_method defined.  And we could apply the same logic here by product.  (although the field is not shown in the views)